[2024-09-10T18:26:41.234+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T18:26:41.253+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: main.etl_process manual__2024-09-10T18:26:37.247426+00:00 [queued]>
[2024-09-10T18:26:41.264+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: main.etl_process manual__2024-09-10T18:26:37.247426+00:00 [queued]>
[2024-09-10T18:26:41.265+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2024-09-10T18:26:41.281+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): etl_process> on 2024-09-10 18:26:37.247426+00:00
[2024-09-10T18:26:41.286+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=4640) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-10T18:26:41.287+0000] {standard_task_runner.py:72} INFO - Started process 4641 to run task
[2024-09-10T18:26:41.288+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'main', 'etl_process', 'manual__2024-09-10T18:26:37.247426+00:00', '--job-id', '315', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpn_o9e9v_']
[2024-09-10T18:26:41.289+0000] {standard_task_runner.py:105} INFO - Job 315: Subtask etl_process
[2024-09-10T18:26:41.343+0000] {task_command.py:467} INFO - Running <TaskInstance: main.etl_process manual__2024-09-10T18:26:37.247426+00:00 [running]> on host 276739585c9a
[2024-09-10T18:26:41.440+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='main' AIRFLOW_CTX_TASK_ID='etl_process' AIRFLOW_CTX_EXECUTION_DATE='2024-09-10T18:26:37.247426+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-09-10T18:26:37.247426+00:00'
[2024-09-10T18:26:41.441+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T18:26:51.743+0000] {logging_mixin.py:190} INFO - 590707
[2024-09-10T18:26:52.195+0000] {logging_mixin.py:190} INFO - root
 |-- house_id: integer (nullable = true)
 |-- latitude: float (nullable = true)
 |-- longitude: float (nullable = true)
 |-- maintenance_year: date (nullable = true)
 |-- square: float (nullable = true)
 |-- population: integer (nullable = true)
 |-- region: string (nullable = true)
 |-- locality_name: string (nullable = true)
 |-- address: string (nullable = true)
 |-- full_address: string (nullable = true)
 |-- communal_service_id: string (nullable = true)
 |-- description: string (nullable = true)
[2024-09-10T18:26:52.798+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-09-10T18:26:52.816+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T18:26:52.816+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=main, task_id=etl_process, run_id=manual__2024-09-10T18:26:37.247426+00:00, execution_date=20240910T182637, start_date=20240910T182641, end_date=20240910T182652
[2024-09-10T18:26:52.850+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T18:26:52.870+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-10T18:26:52.872+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
